libp2p/Common.cpp

```diff
@@ -248,9 +248,9 @@ ostream& operator<<(ostream& _out, NodeIPEndpoint const& _ep)
     return _out;
 }
 
-boost::log::formatting_ostream& operator<<(boost::log::formatting_ostream& _log, const Node& _node)
+ostream& operator<<(ostream& _out, const Node& _node)
```

Comment:
Why the move from the boost logging ostream to std::ostream, don't we only log Node instances using boost loggers? Is the only benefit not having to do static_cast when logging NodeEntry instances?

Reply:
> 
> 
> Why the move from the boost logging ostream to std::ostream, don't we only log Node instances using boost loggers? Is the only benefit not having to do static_cast when logging NodeEntry instances? And out of curiosity, why don't we have to do the static_cast when using std::ostream?

Reply:
Overloading `boost::log::formatting_ostream` is seriously f*cked up. The Node derived types were not accepted and there are issues with `&` and `const&`. This one works for all 4 cases.